torch/_dynamo/variables/tensor.py

```diff
@@ -1062,3 +1064,50 @@ def as_python_constant(self):
 
     def python_type(self):
         return type(self.value)
+
+
+class UntypedStorageVariable(VariableTracker):
+    _nonvar_fields = {
+        "example_value",
+        *VariableTracker._nonvar_fields,
+    }
+
+    def __init__(
+        self,
+        from_tensor: TensorVariable,
+        example_value: torch.UntypedStorage,
+        **kwargs,
+    ):
+        super().__init__(**kwargs),
+        self.from_tensor = from_tensor
+        # Example_value will always have device="meta"
+        self.example_value = example_value
+
+    def call_method(
+        self,
+        tx,
+        name,
+        args: "List[VariableTracker]",
+        kwargs: "Dict[str, VariableTracker]",
+    ) -> "VariableTracker":
+        if name == "size":
+            assert not (args or kwargs)
```

Comment:
split into two separate "assert not"s so that it's easier to see which invariant failed.